core/tests/test_common.py

```diff
@@ -0,0 +1,16 @@
+import geopandas
+import momepy
+
+import core
+
+
+class TestContinuity:
+    def setup_method(self):
+        self.city = "Liège"
+        self.roads = momepy.remove_false_nodes(core.utils.read_parquet_roads(self.city))
```

Comment:
I think that we should simply have another version of our parquet inputs with the nodes of degree 2 removed.

Reply:
good idea!!

Reply:
xref #83